src/overview.rst

```diff
@@ -0,0 +1,328 @@
+          [[[R1]]]
+              graph = "prep => foo"
+          [[[R/^/P1D]]]
+              graph = "foo => bar => baz"
+
+Cylc 8 cleans this up:
+
+.. code-block:: cylc
+
+   [scheduling]
+      initial cycle point = now
+      [[graph]]  # Cylc 8
+          R1 = "prep => foo"
+          R/^/P1D = "foo => bar => baz"
+
+Workflow Installation
+---------------------
+
+The functionality of `rose suite-run` has been migrated into Cylc 8. This
+cleanly separates workflow source directory from run directory, and installs
+workflow files into the run directory at start-up
+
+- `cylc install` copies all workflow source files into a dedicated
+  run-directory
+- each new install creates a new numbered run-directory (by default)
+- (workflow files are automatically installed onto job platforms too)
+
+.. code-block:: bash
+
+   (venv) $ pwd
+   /home/oliverh/cylc-src/democ8
+
+   (venv) $ cylc install
+   INSTALLED democ8 from /home/oliverh/cylc-src/democ8 -> /home/oliverh/cylc-run/democ8/run1
+
+   (venv) $ cylc play democ8/run1
+                ._.
+                | |
+    ._____._. ._| |_____.
+    | .___| | | | | .___|       The Cylc Workflow Engine [8.0b0]
+    | !___| !_! | | !___.           Copyright (C) 2008-2021 NIWA
+    !_____!___. |_!_____!   & British Crown (Met Office) & Contributors.
+          .___! |
+          !_____!
+
+   ...
+
+   (venv) $ cylc install
+   INSTALLED democ8 from /home/oliverh/cylc-src/democ8 -> /home/oliverh/cylc-run/democ8/run2
+
+   (venv) $ cylc play democ8/run2
+   # etc.
+
+Safe Run Semantics
+------------------
+
+Cylc 7 run semantics were dangerous: if you accidentally typed `cylc run`
+instead of `cylc restart` a new from-scratch run would overwrite the existing
+run directory, including the run database, so that you could not go back and do
+the intended restart.
+
+Cylc 8 has `cylc play` to *start*, *restart*, or *unpause* a workflow, so
+"restart" is now the safe default behaviour. For a new run from scratch,
+do a fresh `cylc install` and run it safely in the new run directory.
+
+Security
+--------
+
+- Users authenticate at the Hub, with site-appropriate authentication plugins
+- The Hub spawns a UI Server as the target user (workflow owner). This UI
+  Server interacts with its own schedulers and authorizes access to them
+  according to the privileges granted to the authenticated user
+  - (the UI Server and Schedulers run as the workflow-owner user)
+- Jobs authenticate to their parent scheduler using
+  `CurveZMQ <http://curvezmq.org/>`_
+
+.. note::
+
+   The authorization system is still in development; for the moment you can
+   only interact with your own workflows.
+
+Packaging
+---------
+
+Cylc 7 had to be installed by unpacking a release tarball and ensuring
+that many software dependencies were also installed on the system.
+
+Cylc 8 can be installed from **Conda Forge**, into a conda environment:
+
+.. code-block:: bash
+
+   $ conda create -n cylc8 python=3.8
```

Comment:
Could drop the Python dep here (the forge recipe will pull it in at a compatible version):

```suggestion
   $ conda create -n cylc8
```